src/hermes/commands/workflow.py

```diff
@@ -132,10 +132,17 @@ def process():
 
 
 @click.group(invoke_without_command=True)
-def curate():
+@click.pass_context
+def curate(click_ctx: click.Context):
     ctx = CodeMetaContext()
+    process_output = ctx.hermes_dir / 'process' / 'codemeta.json'
+
+    if not process_output.is_file():
+        click.echo("No processed metadata found. Please run `hermes process` before curation.")
+        click_ctx.exit(69)
```

Comment:
```suggestion
        click_ctx.exit(1)
```

I would suggest sticking with 1 until we have agreed/decided which scenario should produce which exit code.

Reply:
I would prefer 45067 or 2645 as long as #6 is open. But 1 is fine for me, too...